spelling.dic

```diff
@@ -56,9 +58,8 @@ typeof
 uint
 unadvise
 ushort
+Usings
```

Comment:
does this need to be capitalized?

Reply:
Good question. As far as I can tell it doesn't matter, but perhaps it should matter.